.github/workflows/publish.yml

```
@@ -18,10 +18,6 @@ jobs:
           - [ubuntu-latest, manylinux_x86_64]
           - [windows-latest, win_amd64]
           - [macos-latest, macosx_x86_64]
```

```suggestion
          - [macos-latest, macosx_x86_64]
          - [macos-latest, macosx_arm64]
```

I think the issue I wrote was misleading here... we still need to build for arm64, we just don't want to do it with the xlarge runner. This might take a few tries futzing with GitHub CI for it to work. It might also still need to exclude python 3.9... but we should try again without excluding it